### ISSUE TITLE
Monitoring Canopy Height in the Hainan Tropical Rainforest Using Machine Learning and Multi-Modal Data Fusion

### ISSUE THREAD

**Issue by thereisatablehere** [thereisatablehere/CapstoneResources#6]
### Summary 
Biomass carbon sequestration and sink capacities of tropical rainforests are vital for addressing climate change. However, canopy height must be accurately estimated to determine carbon sink potential and implement effective forest management. Four advanced machine-learning algorithms—random forest (RF), gradient boosting decision tree, convolutional neural network, and backpropagation neural network—were compared in terms of forest canopy height in the Hainan Tropical Rainforest National Park. A total of 140 field survey plots and 315 unmanned aerial vehicle photogrammetry plots, along with multi-modal remote sensing datasets (including GEDI and ICESat-2 satellite-carried LiDAR data, Landsat images, and environmental information) were used to validate forest canopy height from 2003 to 2023. The results showed that RH80 was the optimal choice for the prediction model regarding percentile selection, and the RF algorithm exhibited the optimal performance in terms of accuracy and stability, with R2 values of 0.71 and 0.60 for the training and testing sets, respectively, and a relative root mean square error of 21.36%. The RH80 percentile model using the RF algorithm was employed to estimate the forest canopy height distribution in the Hainan Tropical Rainforest National Park from 2003 to 2023, and the canopy heights of five forest types (tropical lowland rainforests, tropical montane cloud forests, tropical seasonal rainforests, tropical montane rainforests, and tropical coniferous forests) were calculated. The study found that from 2003 to 2023, the canopy height in the Hainan Tropical Rainforest National Park showed an overall increasing trend, ranging from 2.95 to 22.02 m. The tropical montane cloud forest had the highest average canopy height, while the tropical seasonal forest exhibited the fastest growth. The findings provide valuable insights for a deeper understanding of the growth dynamics of tropical rainforests.

### Key Points 
- Convolutional neural network
- Hainan Tropical Rainforest National Park
- Multi-modal remote sensing datasets
- Canopy height
- GEDI
- ICESat-2 ATLAS

### Citation
Q. Ling, Y. Chen, Z. Feng, H. Pei, C. Wang, Z. Yin, and Z. Qiu, "Monitoring canopy height in the Hainan tropical rainforest using machine learning and multi-modal data fusion," _Remote Sensing_, vol. 17, no. 6, p. 966, 2025, doi: 10.3390/rs17060966.

### Repo link
N/ A
